logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/VideoStateChecker.kt

```diff
@@ -0,0 +1,15 @@
+package com.wire.kalium.logic.data.call
+
+import com.wire.kalium.calling.VideoStateCalling
+
+interface VideoStateChecker {
+    fun isCameraOn(state: VideoStateCalling): Boolean
+}
+
+class VideoStateCheckerImpl : VideoStateChecker {
+
+    override fun isCameraOn(state: VideoStateCalling) = when (state) {
+        VideoStateCalling.PAUSED, VideoStateCalling.STOPPED -> false
+        else -> true
```

Comment:
Here we are assuming that even if the Video State is one of these: `SCREENSHARE, UNKNOWN` we are still marking it as `true` ? 🤔

Reply:
`SCREENSHARE` to be handled in next PR..for `UNKNOWN` in theory it could not happen but to be safe I will return false for that case